client/src/components/layouts/dashboardLayout/DashboardLayoutRoute.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import DashboardLayout from './DashboardLayout';
+
+const DashboardLayoutRoute = ({ component: Component, ...rest}) => {
+    //get 'component' and renamed to 'Component', any other 'props' renamed to 'rest'
+    return (
+        <Route 
+            {...rest} 
```

Comment:
What does {...rest} do here?

Reply:
My guessing/understanding is that it only uses the `Component` which is the page rendered in the layout, and pass other `props` into `<Route>`  tag. For example when we coded `<DashboardLayoutRoute path="/publications/team" exact component={PublicationPage} />`, `props.component` will go to the `<Component>`, and `props.path` will become part of the `{...rest}` and passed into `<Route>`

Reply:
Ok makes sense